task7/index.js

```diff
@@ -1,17 +1,22 @@
+const obj = [
+    {to: "bookmark.html", img: "1.png", alt:"画像1", text: "ブックマーク"}, 
+    {to: "message.html", img: "2.png", alt:"画像2", text: "メッセージ"}
+]
+
 const div = document.getElementById("js-loading");
 const gif = document.createElement('img');
-gif.src = "loading-circle.gif";
 div.appendChild(gif);
+const loading = () => {
+    gif.src = "loading-circle.gif";
+};
```

Comment:
I don't think there is much point in making it a function.
What we are looking for here is
```js
const div = document.getElementById("js-loading");
const gif = document.createElement('img');
gif.src = "loading-circle.gif";
div.appendChild(gif);
```
The idea is to make the process here a function.

Reply:
I strongly agree.
This function really doesn't make sense.